docs/index.rst

```diff
@@ -714,6 +714,68 @@ method and ``Flask-Classful`` will take care of the rest::
             return "Looking at me? I guess you're logged in."
 
 
+Integrating with webargs
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the wider Flask ecosystem, `webargs <https://webargs.readthedocs.io>`_
+is a commonly-used library to parse and validate HTTP requests. There
+are a couple of things to bear in mind when using ``webargs`` with
+``Flask-Classful``.
+
+If using the `@use_args
+<https://webargs.readthedocs.io/en/latest/api.html#webargs.core.Parser.use_args>`_
+decorator, the parsed arguments are injected into your view. However,
+by default, the args variable would also be included in your route.
+e.g.::
+
+    class UsersView(FlaskView):
+        @use_args(...)
+        def post(self, id, args):
+           # the rest of the view
+
+will result in a route such as ``/users/<id>/<args/`` which is obviously
+not what you want. The workaround is to include ``base_args = ['args`]``
+in your view, which causes `build_rule <http://flask-classful.teracy.org/#api>`_
```

Comment:
@hoatle Firstly, I will add that to the docs now. Secondly, for *ages*, I've been trying to get to the bottom of a weird error I was having, `TypeError: patch() got multiple values for argument 'id'`. I now see that this is because I had `def patch(self, id, args)`, and not `def patch(self, args, id)` (wrong order...). On one hand I'm happy that I learnt a lot on the way about decorators and other things I think I should know, on the other hand I'm super-frustrated that I didn't work it out. As always, hindsight is really helpful. So a big thank you!